src/components/App/MovieDetails/MovieDetails.css

```diff
@@ -0,0 +1,153 @@
+.details-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.background-img {
+    pointer-events: none;
+    position: fixed; 
+    top: 65px;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    filter: blur(15px);
+    transform: scale(1.1);
+    object-fit: cover;
+    z-index: 0;
+}
+
+.top-buttons {
+    display: flex;
+    padding: 40px 30px;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    font-family: 'Lato', sans-serif;
+    font-size: 24px;
+    line-height: 24px;
+    text-decoration: none;
+    
+}
+
+.back-button {
+    display: flex;
+    float: left;
+    position: relative;
+    cursor: pointer;
+    height: auto;
+    align-items: center;
+    color: white;
+    text-decoration: none;
+    transition: transform 0.3s;
+}
+
+.back-button:hover {
+    color: white;
+    transform: scale(1.1);
+}
+
+.next-button {
+    display: flex;
+    float: right;
+    position: relative;
+    cursor: pointer;
+    height: auto;
+    align-items: center;
+    color: white;
+    text-decoration: none;
+    transition: transform 0.3s;
+}
+
+.next-button:hover {
+    color: white;
+    transform: scale(1.1);
+}
+
+.left-arrow-img {
+    display:block;
+    margin-right: 35px;
+    font-size: 16px;
+}
+
+.right-arrow-img {
+    display:block;
+    margin-left: 35px;
+    font-size: 16px;
+}
+
```

Comment:
U have to use one class for both buttons and one class for both images cause they are the same